app/Infrastructure/Controllers/GetUserController.php

```diff
@@ -13,9 +13,15 @@ class GetUserController extends BaseController
 
     public function __invoke(string $userId): JsonResponse
     {
+        if(strlen($userId)<1){
```

Comment:
Please, if you have created this piece of code in another class, delete it as soon as possible.

Reply:
I have fixed right now madam